src/components/global/GNB.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from '@emotion/styled';
+import THEME from 'constants/Colors';
+import { useRouter } from 'next/router';
+import React from 'react';
+import { Mypage, TodoBox, TodoMall } from './Icons';
+
+type Props = {};
+
+const GNB = (props: Props) => {
+  const router = useRouter();
+
+  const handleClick = (route: string) => {
```

Comment:
route 보다 어디로 routing이 되는지 알 수 있는 파라미터 값이 좋지 않을까요?

```suggestion
  const handleClick = (path: string) => {
```